doc/main/reference/task_group_bypass_support.rst

```diff
@@ -0,0 +1,94 @@
+.. _task_group_bypass_support:
+
+Task Bypass Support for ``task_group``
+======================================
+
+.. note::
+    To enable this extension, define the ``TBB_PREVIEW_TASK_GROUP_EXTENSIONS`` macro with a value of ``1``.
+
+.. contents::
+    :local:
+    :depth: 2
+
+Description
+***********
+
+The |full_name| implementation extends the requirements for user-provided function object from
+`tbb::task_group specification <https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/task_scheduler/task_group/task_group_cls>`_
+to allow them to return a ``task_handle`` object.
```

Comment:
It seems that [task_group specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/task_scheduler/task_group/task_group_cls) is silent on the return type of the user-provided function object.